src/librustc_target/spec/riscv32imac_unknown_none_elf.rs

```diff
@@ -25,11 +25,11 @@ pub fn target() -> TargetResult {
         linker_flavor: LinkerFlavor::Ld,
 
         options: TargetOptions {
-            linker: Some("riscv32-unknown-elf-ld".to_string()),
+            linker: Some("rust-lld".to_string()),
             cpu: "generic-rv32".to_string(),
             max_atomic_width: Some(32),
             atomic_cas: false, // incomplete +a extension
```

Comment:
Just checking: this hasn't been fixed yet?

Reply:
Nope, a few commits have happened but nothing that "interesting".